### PR TITLE
Write level 31 instead of 0.

### PR DESCRIPTION
**Pull request by tdaede**
We can't guarantee we meet a level, so choose level 31 for now
which has no constraints.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -605,7 +605,8 @@ impl<'a> UncompressedHeader for BitWriter<'a, BE> {
         self.write_bit(false)?; // no timing info present
         self.write(5, 0)?; // one operating point
         self.write(12,0)?; // idc
-        self.write(5, 0)?; // level
+        self.write(5, 31)?; // level
+        self.write(1, 0)?; // tier
```
Comment (rzumer) [inline src/lib.rs]: Seems that tier is not written to the header.

Comment (tdaede) [reply src/lib.rs]: The condition is actually whether level is >7:
https://aomediacodec.github.io/av1-spec/#sequence-header-obu-syntax

Comment (rzumer) [reply src/lib.rs]: I see. I was looking at the reduced still picture header area, my bad.

Comment (rzumer) [reply src/lib.rs]: I see. I was looking at the reduced still picture block, which does not code tier values in any case. My bad.

**Pull request merged**